### PR TITLE
Jhalackova deploying raw images

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Describe the overall goals of this pull request.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

File: tasks/deployment-raw-procedure.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      <para>
+        Without the configuration device, the <emphasis>&jeosfirstboot;</emphasis> is triggered to perform minimal
+        configuration. For details, refer to <xref linkend="deployment-jeos-firstboot"/>.
+      </para>
+    </step>
+    <step>
+      <para>
+        After successful deployment, register your system as described in.
+      </para>
+    </step>
```
Comment (tbazant) [inline tasks/deployment-raw-procedure.xml:73]: I would remove this step and append it to the end of Firstboot, that makes the transiotion from this to Firstboot more streamlined
```suggestion
    
```

File: tasks/deployment-jeos-firstboot.xml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        </textobject>
+      </mediaobject>
+    </figure>
+  </step>
+  <step>
+    <para>
+      &productnameshort; is successfully deployed using a minimal initial
+      configuration.
+    </para>
+  </step>
```
Comment (tbazant) [inline tasks/deployment-jeos-firstboot.xml:155]:
```suggestion
   <step>
      <para>
        After successful deployment, register your system as described in.
      </para>
    </step>
```

**Review by tbazant — CHANGES_REQUESTED** [review]
Great job Jana, consider my suggestions pls, hope they make sense

File: tasks/deployment-raw-procedure.xml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    <step>
+      <para>
+        The raw image comes with a default partitioning scheme (described in <xref linkend="deployment-default-partitioning"/>). If you want to change it or perform
+        complex configuration task, you need to prepare the configuration device as described in <xref
+        linkend="deployment-preparing-configuration-device"/>. Otherwise, you can proceed to the next step.
+      </para>
+    </step>
+    <step>
+      <para>
+        Boot the raw image disk and in the boot loader screen select <guimenu>SLE Micro
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure.xml:59]:
```suggestion
        Boot the raw disk image, and in the boot loader screen, select <guimenu>SLE Micro
```

File: tasks/deployment-preparing-configuration-device.xml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        Set the device label to either <literal>ignition</literal> (when either
+    Ignition or Combustion is used) or <literal>combustion</literal> (when only
+    Combustion is used). If needed (for example on Windows host), use uppercase letters for the labels. To label the device, run:
+      </para>
+      <screen>
+&prompt.user;sudo e2label /dev/sd<replaceable>Y</replaceable> ignition
+      </screen>
+      <para>
+        You can use any type of configuration storage media that your
+        virtualization system or your hardware supports: ISO image, a USB
```
Comment (dariavladykina) [inline tasks/deployment-preparing-configuration-device.xml:53]:
```suggestion
        virtualization system or your hardware supports: an ISO image, a USB
```

File: tasks/deployment-jeos-firstboot.xml [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <info>
+    <title>Configuring &productnameshort; with &jeosfirstboot;</title><!-- can be changed via merge
+in the assembly -->
+    <!-- add author's e-mail -->
+    <meta name="maintainer" content="jsindelarova@suse.com" its:translate="no"/>
+    <abstract>
+      <para>
+        When booting the &productnameshort; for the first time without providing any configuration device,
```
Comment (dariavladykina) [inline tasks/deployment-jeos-firstboot.xml:25]:
```suggestion
        When booting &productnameshort; for the first time without providing any configuration device,
```

File: tasks/deployment-registering-system.xml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+      &productname; is now registered.
+     </para>
+    </step>
+   </procedure>
+   <note>
+    <title>Other registration options</title>
+  <para>
+     For information
+    that goes beyond the scope of this section, refer to the inline
+    documentation with <command>SUSEConnect --help</command>
```
Comment (dariavladykina) [inline tasks/deployment-registering-system.xml:75]:
```suggestion
    documentation with <command>SUSEConnect --help</command>.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Jana, please find some comments here. Thanks!

**Pull request merged**